parser.py

```diff
@@ -73,7 +73,7 @@ def make_message(cls, doc_uuid):
     def load_file(cls, doc_uuid, tsv_file):
         if not (bytes is str):
             tsv_file = str(tsv_file, 'utf-8')
-        lines = [ x.strip() for x in tsv_file.splitlines() if x.strip() ]
+        lines = ( x.strip() for x in tsv_file.splitlines() if x.strip() )
```

Comment:
I agree. As the size of a tsv file can be huge and we iterate lines once, generator is much better than list comprehension.